src/server/api-server.js

```diff
@@ -77,7 +78,7 @@ class APIServer extends EventEmitter {
   startListening() {
     return new Promise((resolve, reject) => {
       this._expressServer = this._expressApp.listen(
-        this._port, this._host, () => {
+        this._port, this._serveHost, () => {
```

Comment:
Am I right in assuming that '0.0.0.0' wasn't allowed in those places you've references?

*Note they aren't security exceptions, it just manually setting the permission model for notifications for a specific page